data-reporting-service/organization-service/src/main/java/gov/cdc/etldatapipeline/organization/model/dto/org/OrganizationSp.java

```diff
@@ -0,0 +1,79 @@
+package gov.cdc.etldatapipeline.organization.model.dto.org;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import gov.cdc.etldatapipeline.organization.utils.DataPostProcessor;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import lombok.*;
+
+/**
+ * Data Model to capture the results of the stored procedure `sp_organization_event`
+ */
+@Data
+@Builder
+@Entity
+@AllArgsConstructor
+@NoArgsConstructor
+@ToString
```

Comment:
If you're adding `@Data` annotation, that will take care of `@ToString` as well.

Reply:
yes. Good catch. Forgot to remove it.  Will remove the `@ToString`